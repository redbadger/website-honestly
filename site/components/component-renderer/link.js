// @flow
import * as React from 'react';
import styles from './styles.css';

type LinkProps = {
  children?: React.Node,
  href: string
}

export default function Link({ children, href }: LinkProps) {
  return (
    <a className={styles.a} href={href}>
      {children}
    </a>
  );
}
