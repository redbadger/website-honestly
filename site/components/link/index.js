// @flow
import React, { PropTypes } from 'react';
import { NavigationLink } from 'navigation-react';

type LinkProps = {[id: string]: any}
export default function Link(props: LinkProps) {
  const linkProps = Object.assign({}, props, {
    stateKey: props.to,
  });

  return (
    <NavigationLink
      {...linkProps}
    >
      {props.children}
    </NavigationLink>
  );
}

Link.propTypes = {
  to: PropTypes.string.isRequired,
  children: PropTypes.node.isRequired,
};
