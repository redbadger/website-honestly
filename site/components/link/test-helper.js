// @flow

import { StateNavigator, HTML5HistoryManager } from 'navigation';
import * as React from 'react';
import { NavigationHandler } from 'navigation-react';
import makeAppNavigator from '../../routes';

export function mockNavigator(parentKey?: string = 'foo') {
  // The constructor is checked in prop-types,
  // so we use it as base and then mutate

  const navigator = new StateNavigator(
    [
      { key: 'foo', route: 'foo' },
      { key: 'bar', route: 'bar' },
      { key: 'barChild', route: 'bar/child', parentKey: 'bar' },
    ],
    new HTML5HistoryManager(),
  );

  navigator.navigate(parentKey);

  return navigator;
}

type Props = {
  stateNavigator?: StateNavigator,
  children: React.Node,
};

const initStateNavigator = () => {
  const navigator = makeAppNavigator();

  navigator.navigateLink('/');

  return navigator;
};

export const Context = (props: Props) => {
  const { children, stateNavigator = initStateNavigator() } = props;

  return <NavigationHandler stateNavigator={stateNavigator}>{children}</NavigationHandler>;
};
