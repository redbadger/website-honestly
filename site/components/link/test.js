// @flow
import React from 'react';
import { shallow } from 'enzyme';
import Link from '.';


describe('components/link', () => {
  it('renders OK children', () => {
    shallow(<Link to="foo">Hello</Link>);
  });
});
