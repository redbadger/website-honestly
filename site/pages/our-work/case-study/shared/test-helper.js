// @no-flow
import React from 'react';

import styles from './what-to-read-next/style.css';
import Link from '../../../../components/link';
import WhatToReadNextSlice from './what-to-read-next/slice';

const MockedWhatToReadNext = () => {
  const nextSlices = [
    {
      name: 'Pride in London',
      tagline: 'Delivering a best-in-class digital experience',
      image: 'pride.gif',
      linkKey: 'prideCaseStudy',
    },
    {
      name: 'Financial Times',
      tagline: 'Lasting change for a media giant',
      image: 'financialTimesReadMore.jpg',
      linkKey: 'financialTimesCaseStudy',
    },
    {
      name: 'Car Trawler',
      tagline: 'Creating complete CMS control for travel brands',
      image: 'carTrawler.jpg',
      linkKey: 'carTrawlerCaseStudy',
    },
  ];
  return (
    <div className={styles.whatNext}>
      <div className={styles.whatNext__tilesContainer}>
        {nextSlices.map((slice, index) => (
          /* disabled because list is static */
          /* eslint-disable react/no-array-index-key */
          <WhatToReadNextSlice
            key={`${slice.name}-${index}`}
            name={slice.name}
            tagline={slice.tagline}
            image={slice.image}
            linkKey={slice.linkKey}
          />
          /* eslint-enable react/no-array-index-key */
        ))}
      </div>
      <Link to="ourWorkPage" className={styles.whatNext__button}>
        See more of our work
      </Link>
    </div>
  );
};

export default MockedWhatToReadNext;
