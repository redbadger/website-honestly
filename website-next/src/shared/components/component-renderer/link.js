import React, { Component } from 'react';
import styles from './styles.css';

export default class Link extends Component {
  static propTypes = {
    children: React.PropTypes.node,
    href: React.PropTypes.string.isRequired,
  };

  render() {
    return (
      <a className={styles.a} href={this.props.href}>{this.props.children}</a>
    );
  }
}
