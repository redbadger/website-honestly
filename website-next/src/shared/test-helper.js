import TestUtils from 'react-addons-test-utils';
import equals from 'lodash/isEqual';
import ShallowTestUtils from 'react-shallow-testutils';

export function render(component) {
  const renderer = TestUtils.createRenderer();
  renderer.render(component);
  return renderer.getRenderOutput();
}

export function hasProp(type, value) {
  return (component) => component.props && component.props[type] === value;
}

export function isEqual(value) {
  return (component) => equals(value, component);
}

export function findOne(predicate, component) {
  const elem = ShallowTestUtils.findAll(component, predicate);

  if (elem.length !== 1) {
    throw new Error('Did not find exactly one match');
  }

  return elem[0];
}

export function containsOne(value, component) {
  return !!findOne(isEqual(value), component);
}

export function findWithType(type, component) {
  return ShallowTestUtils.findWithType(component, type);
}

export function findAllWithType(type, component) {
  return ShallowTestUtils.findAllWithType(component, type);
}
