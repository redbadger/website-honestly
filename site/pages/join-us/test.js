import React from 'react';
import { mount } from 'enzyme';

import { Context } from '../../components/link/test-helper';

import JoinUs from './index';

const jobs = [
  {
    id: '1',
    title: 'chief awesome person',
    description: 'awesomeness is a full-time job so we need you here being awesome',
    fullDescription:
      'awesomeness is a full-time job so we need you here being awesome and that is all I have to say about that',
    department: 'Awesome Squad',
    slug: 'awesome',
    applicationUrl: 'https://www.red-badger.com/jobs/awesome',
    datePosted: 'Thu Feb 08 2018 16:42:31 GMT+0000 (Greenwich Mean Time)',
  },
  {
    id: '2',
    title: 'Junior adorable dog',
    description:
      'Being (mostly) the only aodrable dog in the office is hard work for milo, so we need an apprentice to take up the slack',
    fullDescription:
      'Being (mostly) the only aodrable dog in the office is hard work for milo, so we need an apprentice to take up the slack and that is all I have to say about that',
    department: 'four legged friends',
    slug: 'woof',
    applicationUrl: 'https://www.red-badger.com/jobs/woof',
    datePosted: 'Thu Feb 08 2018 16:42:31 GMT+0000 (Greenwich Mean Time)',
  },
  {
    id: '3',
    title: 'Mid 2 Tea drinker',
    description:
      'Ideally you have 2-4 years experience drinking tea, proficiency with green and white teas is ideal but not required',
    fullDescription:
      'Ideally you have 2-4 years experience drinking tea, proficiency with green and white teas is ideal but not required and that is all I have to say about that',
    department: 'Caffiends',
    slug: 'slurp',
    applicationUrl: 'https://www.red-badger.com/jobs/tea',
    datePosted: 'Thu Feb 08 2018 16:42:31 GMT+0000 (Greenwich Mean Time)',
  },
];

describe('site/pages/join-us', () => {
  it('renders correctly', () => {
    expect(
      mount(
        <Context>
          <JoinUs jobs={jobs} />
        </Context>,
      ),
    ).toMatchSnapshot();
  });
});
