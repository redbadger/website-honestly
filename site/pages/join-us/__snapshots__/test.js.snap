// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`site/pages/join-us renders correctly 1`] = `
<div
  class="background"
>
  <div
    class="jobs"
  >
    <div
      class="container"
    >
      <div
        class="headings"
      >
        <h1
          class="h1"
        >
          Join us
        </h1>
        <p>
          Our people are the best thing about Red Badger. Do you love solving difficult problems? Do you enjoy working with great people? Do you want to have a real impact? Join us and become a Badger.
        </p>
      </div>
    </div>
  </div>
  <div
    class="wrapper"
  >
    <div
      class="container"
    >
      <div
        class="galleryContainer"
      >
        <div
          class="galleryWrapper"
        >
          <div
            class="dodgeballPhotoWrapper hoverHandler"
          >
            <button
              class="dodgeballButton"
              type="button"
            >
              <div
                class="dodgeballPhotoMask"
              >
                <div
                  class="illustrationWrapper"
                >
                  <img
                    alt="Our Red Badger Dodgeball team ready for the next game."
                    class="illustrationWide"
                    src="dodgeball_illustrated.jpg"
                  />
                </div>
                <div
                  class="photoWrapper "
                >
                  <img
                    alt="Our Red Badger Dodgeball team ready for the next game."
                    class="photoWide"
                    src="dodgeball_photo.jpg"
                  />
                </div>
              </div>
              <img
                alt=""
                class="dodgeballFrame"
                src="dodgeball_frame.png"
              />
            </button>
          </div>
          <div
            class="scientistPhotoWrapper hoverHandler"
          >
            <button
              class="scientistButton"
              type="button"
            >
              <div
                class="scientistPhotoMask"
              >
                <div
                  class="illustrationWrapper"
                >
                  <img
                    alt="Stuart Harris, Red Badger co-founder and chief scientist in a lab coat reading a report."
                    class="illustration"
                    src="scientist_illustrated.jpg"
                  />
                </div>
                <div
                  class="photoWrapper "
                >
                  <img
                    alt="Stuart Harris, Red Badger co-founder and chief scientist in a lab coat reading a report."
                    class="photo"
                    src="scientist_photo.jpg"
                  />
                </div>
              </div>
              <img
                alt=""
                class="scientistFrame"
                src="scientist_frame.png"
              />
            </button>
          </div>
          <div
            class="anthonyNolanPhotoWrapper hoverHandler"
          >
            <button
              class="anthonyNolanButton"
              type="button"
            >
              <div
                class="anthonyNolanPhotoMask"
              >
                <div
                  class="illustrationWrapper"
                >
                  <img
                    alt="People testing a prototype of a new tool."
                    class="illustration"
                    src="anthonynolan_illustrated.jpg"
                  />
                </div>
                <div
                  class="photoWrapper photoWrapperAdjustment"
                >
                  <img
                    alt="People testing a prototype of a new tool."
                    class="photo"
                    src="anthonynolan_photo.jpg"
                  />
                </div>
              </div>
              <img
                alt=""
                class="anthonyNolanFrame"
                src="anthonynolan_frame.png"
              />
            </button>
          </div>
          <div
            class="pridePhotoWrapper hoverHandler"
          >
            <button
              class="prideButton"
              type="button"
            >
              <div
                class="pridePhotoMask"
              >
                <div
                  class="illustrationWrapper"
                >
                  <img
                    alt="A group of people brainstorming new ideas in a workshop."
                    class="illustrationWide"
                    src="pride_illustrated.jpg"
                  />
                </div>
                <div
                  class="photoWrapper "
                >
                  <img
                    alt="A group of people brainstorming new ideas in a workshop."
                    class="photoWide"
                    src="pride_photo.jpg"
                  />
                </div>
              </div>
              <img
                alt=""
                class="prideFrame"
                src="pride_frame.png"
              />
            </button>
          </div>
          <div
            class="sketchSessionPhotoWrapper hoverHandler"
          >
            <button
              class="sketchSessionButton"
              type="button"
            >
              <div
                class="sketchSessionPhotoMask"
              >
                <div
                  class="illustrationWrapper"
                >
                  <img
                    alt="A team sitting at a meeting room table."
                    class="illustrationWide"
                    src="sketch_session_illustrated.jpg"
                  />
                </div>
                <div
                  class="photoWrapper "
                >
                  <img
                    alt="A team sitting at a meeting room table."
                    class="photoWide"
                    src="sketch_session_photo.jpg"
                  />
                </div>
              </div>
              <img
                alt=""
                class="sketchSessionFrame"
                src="sketch_session_frame.png"
              />
            </button>
          </div>
          <div
            class="janPhotoWrapper hoverHandler"
          >
            <button
              class="janButton"
              type="button"
            >
              <div
                class="janPhotoMask"
              >
                <div
                  class="illustrationWrapper"
                >
                  <img
                    alt="People celebrating at the Pride parade./ A young man holding up a big promotional poster at the Pride Parade."
                    class="illustration"
                    src="jan_illustrated.jpg"
                  />
                </div>
                <div
                  class="photoWrapper "
                >
                  <img
                    alt="People celebrating at the Pride parade./ A young man holding up a big promotional poster at the Pride Parade."
                    class="photo"
                    src="jan_photo.jpg"
                  />
                </div>
              </div>
              <img
                alt=""
                class="janFrame"
                src="jan_frame.png"
              />
            </button>
          </div>
          <div
            class="miloPhotoWrapper hoverHandler"
          >
            <button
              class="miloButton"
              type="button"
            >
              <div
                class="miloPhotoMask"
              >
                <div
                  class="illustrationWrapper"
                >
                  <img
                    alt="Our office dog and a Test Lead working on a laptop."
                    class="illustrationWide"
                    src="milo_illustrated.jpg"
                  />
                </div>
                <div
                  class="photoWrapper "
                >
                  <img
                    alt="Our office dog and a Test Lead working on a laptop."
                    class="photoWide"
                    src="milo_photo.jpg"
                  />
                </div>
              </div>
              <img
                alt=""
                class="miloFrame"
                src="milo_frame.png"
              />
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="wrapper"
  >
    <div
      class="container"
    >
      <div
        class="jobs jobs"
      >
        <div
          class="jobsListTimelineSection"
        >
          <h1
            class="jobsTimelineTitle"
          >
            We’re hiring
          </h1>
          <div
            class="jobsTimelineIntro"
          >
            <p>
              We believe in creating opportunity for all. We benefit from diversity. That said, we're far from perfect – we want to be more diverse and representative. Join us and help make that happen.
            </p>
          </div>
          <hr
            class="hr black horizontal-line"
          />
          <div
            class="jobSectionContainer"
          >
            <h2
              class="jobSectionHeader"
            >
              Awesome Squad
            </h2>
            <ul
              class="jobsList"
            >
              <li
                class="jobItem"
                itemscope=""
                itemtype="http://schema.org/JobPosting"
              >
                <div
                  class="jobEntryWrapper"
                >
                  <div
                    itemprop="jobLocation"
                    itemscope=""
                    itemtype="http://schema.org/Place"
                  >
                    <meta
                      content="4th Floor, 2 Old Street Yard"
                      itemprop="streetAddress"
                    />
                    <meta
                      content="London"
                      itemprop="addressLocality"
                    />
                    <meta
                      content="Greater London"
                      itemprop="addressRegion"
                    />
                    <meta
                      content="EC1Y 8AF"
                      itemprop="postalCode"
                    />
                  </div>
                  <meta
                    content="Red Badger"
                    itemprop="hiringOrganization"
                  />
                  <meta
                    content="2018-02-08T16:42:31.000Z"
                    itemprop="datePosted"
                  />
                  <div
                    class="grid withGutter responsive-grid-smallScreen"
                  >
                    <div
                      class="responsive-cell-smallScreen cell size12of12 hideOndont"
                    >
                      <button
                        aria-expanded="false"
                        aria-label="expand chief awesome person job description"
                        class="jobEntry__more"
                        type="button"
                      >
                        <span
                          class="jobEntry__arrow"
                        />
                      </button>
                      <div
                        class="jobEntryHeaderContainer"
                      >
                        <div
                          class="jobEntryHeader"
                          itemprop="title"
                        >
                          <h3
                            class="jobTitle"
                            itemprop="name"
                          >
                            <span>
                              chief awesome person
                            </span>
                          </h3>
                        </div>
                        <div
                          class="jobDescription__hidden"
                        >
                          <div
                            class="jobDescription"
                            itemprop="description"
                          >
                            <div
                              class="typography rawHtml"
                            >
                              awesomeness is a full-time job so we need you here being awesome
                            </div>
                          </div>
                          <a
                            class="applyBtn"
                            href="/jobs/awesome"
                          >
                            Read more
                          </a>
                        </div>
                      </div>
                      <hr
                        class="hr black mobileHorizontalLine"
                      />
                    </div>
                  </div>
                </div>
              </li>
            </ul>
          </div>
          <hr
            class="hr black horizontal-line"
          />
          <div
            class="jobSectionContainer"
          >
            <h2
              class="jobSectionHeader"
            >
              four legged friends
            </h2>
            <ul
              class="jobsList"
            >
              <li
                class="jobItem"
                itemscope=""
                itemtype="http://schema.org/JobPosting"
              >
                <div
                  class="jobEntryWrapper"
                >
                  <div
                    itemprop="jobLocation"
                    itemscope=""
                    itemtype="http://schema.org/Place"
                  >
                    <meta
                      content="4th Floor, 2 Old Street Yard"
                      itemprop="streetAddress"
                    />
                    <meta
                      content="London"
                      itemprop="addressLocality"
                    />
                    <meta
                      content="Greater London"
                      itemprop="addressRegion"
                    />
                    <meta
                      content="EC1Y 8AF"
                      itemprop="postalCode"
                    />
                  </div>
                  <meta
                    content="Red Badger"
                    itemprop="hiringOrganization"
                  />
                  <meta
                    content="2018-02-08T16:42:31.000Z"
                    itemprop="datePosted"
                  />
                  <div
                    class="grid withGutter responsive-grid-smallScreen"
                  >
                    <div
                      class="responsive-cell-smallScreen cell size12of12 hideOndont"
                    >
                      <button
                        aria-expanded="false"
                        aria-label="expand Junior adorable dog job description"
                        class="jobEntry__more"
                        type="button"
                      >
                        <span
                          class="jobEntry__arrow"
                        />
                      </button>
                      <div
                        class="jobEntryHeaderContainer"
                      >
                        <div
                          class="jobEntryHeader"
                          itemprop="title"
                        >
                          <h3
                            class="jobTitle"
                            itemprop="name"
                          >
                            <span>
                              Junior adorable dog
                            </span>
                          </h3>
                        </div>
                        <div
                          class="jobDescription__hidden"
                        >
                          <div
                            class="jobDescription"
                            itemprop="description"
                          >
                            <div
                              class="typography rawHtml"
                            >
                              Being (mostly) the only aodrable dog in the office is hard work for milo, so we need an apprentice to take up the slack
                            </div>
                          </div>
                          <a
                            class="applyBtn"
                            href="/jobs/woof"
                          >
                            Read more
                          </a>
                        </div>
                      </div>
                      <hr
                        class="hr black mobileHorizontalLine"
                      />
                    </div>
                  </div>
                </div>
              </li>
            </ul>
          </div>
          <hr
            class="hr black horizontal-line"
          />
          <div
            class="jobSectionContainer"
          >
            <h2
              class="jobSectionHeader"
            >
              Caffiends
            </h2>
            <ul
              class="jobsList"
            >
              <li
                class="jobItem"
                itemscope=""
                itemtype="http://schema.org/JobPosting"
              >
                <div
                  class="jobEntryWrapper"
                >
                  <div
                    itemprop="jobLocation"
                    itemscope=""
                    itemtype="http://schema.org/Place"
                  >
                    <meta
                      content="4th Floor, 2 Old Street Yard"
                      itemprop="streetAddress"
                    />
                    <meta
                      content="London"
                      itemprop="addressLocality"
                    />
                    <meta
                      content="Greater London"
                      itemprop="addressRegion"
                    />
                    <meta
                      content="EC1Y 8AF"
                      itemprop="postalCode"
                    />
                  </div>
                  <meta
                    content="Red Badger"
                    itemprop="hiringOrganization"
                  />
                  <meta
                    content="2018-02-08T16:42:31.000Z"
                    itemprop="datePosted"
                  />
                  <div
                    class="grid withGutter responsive-grid-smallScreen"
                  >
                    <div
                      class="responsive-cell-smallScreen cell size12of12 hideOndont"
                    >
                      <button
                        aria-expanded="false"
                        aria-label="expand Mid 2 Tea drinker job description"
                        class="jobEntry__more"
                        type="button"
                      >
                        <span
                          class="jobEntry__arrow"
                        />
                      </button>
                      <div
                        class="jobEntryHeaderContainer"
                      >
                        <div
                          class="jobEntryHeader"
                          itemprop="title"
                        >
                          <h3
                            class="jobTitle"
                            itemprop="name"
                          >
                            <span>
                              Mid 2 Tea drinker
                            </span>
                          </h3>
                        </div>
                        <div
                          class="jobDescription__hidden"
                        >
                          <div
                            class="jobDescription"
                            itemprop="description"
                          >
                            <div
                              class="typography rawHtml"
                            >
                              Ideally you have 2-4 years experience drinking tea, proficiency with green and white teas is ideal but not required
                            </div>
                          </div>
                          <a
                            class="applyBtn"
                            href="/jobs/slurp"
                          >
                            Read more
                          </a>
                        </div>
                      </div>
                      <hr
                        class="hr black mobileHorizontalLine"
                      />
                    </div>
                  </div>
                </div>
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </div>
  <section
    class="benefitsTimelineSection"
    id="contactUs"
  >
    <div
      class="wrapper"
    >
      <div
        class="container"
      >
        <div
          class="benefits__wrapper"
        >
          <div
            class="benefits__container"
          >
            <h1
              class="benefits__heading"
            >
              Benefits
            </h1>
            <ul
              class="benefits__categoryList"
            >
              <li>
                <div
                  class="category"
                >
                  <div
                    class="category__content"
                  >
                    <h3
                      class="category__title"
                    >
                      Health
                    </h3>
                    <img
                      alt="Health icon"
                      class="category__icon"
                      src="Health_icon.png"
                    />
                  </div>
                  <ul
                    class="category__questionList category__visible"
                  >
                    <li>
                      <div>
                        <div
                          class="benefit__question"
                        >
                          <h4
                            class="benefit__heading"
                          >
                            How does Red Badger support me if I’m unwell?
                          </h4>
                        </div>
                        <div
                          class="benefit__answer__visible"
                        >
                          <p>
                            As well as enhanced sick pay, we also offer private medical cover provided by Vitality.
                          </p>
                        </div>
                      </div>
                    </li>
                    <li>
                      <div>
                        <div
                          class="benefit__question"
                        >
                          <h4
                            class="benefit__heading"
                          >
                            What is covered to keep me healthy?
                          </h4>
                        </div>
                        <div
                          class="benefit__answer__visible"
                        >
                          <p>
                            We have a cycle to work scheme, which offers a loan of up to £1,000 to lease you a bike and accessories. Our HQ office also has great bike storage and shower facilities. We have ad-hoc yoga and other classes and there’s always a supply of free fruit in our kitchens.
                          </p>
                        </div>
                      </div>
                    </li>
                    <li>
                      <div>
                        <div
                          class="benefit__question"
                        >
                          <h4
                            class="benefit__heading"
                          >
                            In the worst-case scenario, how do Red Badger support me and my family?
                          </h4>
                        </div>
                        <div
                          class="benefit__answer__visible"
                        >
                          <p>
                            We’ve got several benefits designed to help you through difficult times. These include confidential legal advice and counselling through our Employee Assistance Programme, our death in service cover, permanent health insurance and critical illness cover.
                          </p>
                        </div>
                      </div>
                    </li>
                  </ul>
                </div>
              </li>
              <li>
                <div
                  class="category"
                >
                  <div
                    class="category__content"
                  >
                    <h3
                      class="category__title"
                    >
                      Finance
                    </h3>
                    <img
                      alt="Finance icon"
                      class="category__icon"
                      src="Finance_icon.png"
                    />
                  </div>
                  <ul
                    class="category__questionList category__visible"
                  >
                    <li>
                      <div>
                        <div
                          class="benefit__question"
                        >
                          <h4
                            class="benefit__heading"
                          >
                            What pension contributions will I receive?
                          </h4>
                        </div>
                        <div
                          class="benefit__answer__visible"
                        >
                          <p>
                            From day one you’ll be auto-enrolled on our pension scheme and on passing probation you’ll be able to opt-in to our enhanced scheme, which offers an employer contribution of 5% of your salary.
                          </p>
                        </div>
                      </div>
                    </li>
                    <li>
                      <div>
                        <div
                          class="benefit__question"
                        >
                          <h4
                            class="benefit__heading"
                          >
                            Do you offer assistance with the cost of commuting?
                          </h4>
                        </div>
                        <div
                          class="benefit__answer__visible"
                        >
                          <p>
                            Yes, you can take out an interest-free loan to help with the cost of travel to and from work.
                          </p>
                        </div>
                      </div>
                    </li>
                    <li>
                      <div>
                        <div
                          class="benefit__question"
                        >
                          <h4
                            class="benefit__heading"
                          >
                            Is there a bonus scheme?
                          </h4>
                        </div>
                        <div
                          class="benefit__answer__visible"
                        >
                          <p>
                            While we don’t believe in performance-related bonuses, we do have a generous referral bonus scheme.
                          </p>
                        </div>
                      </div>
                    </li>
                  </ul>
                </div>
              </li>
              <li>
                <div
                  class="category"
                >
                  <div
                    class="category__content"
                  >
                    <h3
                      class="category__title"
                    >
                      Work-life balance
                    </h3>
                    <img
                      alt="Work-life balance icon"
                      class="category__icon"
                      src="Worklifebalance_icon.png"
                    />
                  </div>
                  <ul
                    class="category__questionList category__visible"
                  >
                    <li>
                      <div>
                        <div
                          class="benefit__question"
                        >
                          <h4
                            class="benefit__heading"
                          >
                            How much holiday will I get?
                          </h4>
                        </div>
                        <div
                          class="benefit__answer__visible"
                        >
                          <p>
                            In addition to bank holidays, we offer 24 holidays per year, which incrementally increases to 28 days after six years service.
                          </p>
                        </div>
                      </div>
                    </li>
                    <li>
                      <div>
                        <div
                          class="benefit__question"
                        >
                          <h4
                            class="benefit__heading"
                          >
                            What’s your parental leave policy?
                          </h4>
                        </div>
                        <div
                          class="benefit__answer__visible"
                        >
                          <p>
                            Starting at 26 weeks and depending on your length of service, you’ll be eligible for either our enhanced - 12 weeks full pay - or our enhanced plus - 16 weeks full pay - maternity or shared parental leave. New fathers can also take enhanced paternity leave in addition to shared parental leave.
                          </p>
                        </div>
                      </div>
                    </li>
                    <li>
                      <div>
                        <div
                          class="benefit__question"
                        >
                          <h4
                            class="benefit__heading"
                          >
                            Do you have any wellbeing programmes?
                          </h4>
                        </div>
                        <div
                          class="benefit__answer__visible"
                        >
                          <p>
                            Once a month a masseuse comes into the office to give at-desk massages. We also have several wellbeing programmes and talks throughout the year.
                          </p>
                        </div>
                      </div>
                    </li>
                  </ul>
                </div>
              </li>
              <li>
                <div
                  class="category"
                >
                  <div
                    class="category__content"
                  >
                    <h3
                      class="category__title"
                    >
                      Community and culture
                    </h3>
                    <img
                      alt="Community and culture icon"
                      class="category__icon"
                      src="Social_icon.png"
                    />
                  </div>
                  <ul
                    class="category__questionList category__visible"
                  >
                    <li>
                      <div>
                        <div
                          class="benefit__question"
                        >
                          <h4
                            class="benefit__heading"
                          >
                            What socials can I get involved in?
                          </h4>
                        </div>
                        <div
                          class="benefit__answer__visible"
                        >
                          <p>
                            When you join, check out our social calendar to see what’s going on - from dodgeball to pub quizzes to ring making to diversity lunches, you are invited to take part in as much or as little as you would like. And don’t forget our two annual parties!
                          </p>
                        </div>
                      </div>
                    </li>
                    <li>
                      <div>
                        <div
                          class="benefit__question"
                        >
                          <h4
                            class="benefit__heading"
                          >
                            Can I donate to charity through the payroll?
                          </h4>
                        </div>
                        <div
                          class="benefit__answer__visible"
                        >
                          <p>
                            Yes, you can join our Give As You Earn scheme to make tax-free charity donations, Red Badger will also match your donation!
                          </p>
                        </div>
                      </div>
                    </li>
                    <li>
                      <div>
                        <div
                          class="benefit__question"
                        >
                          <h4
                            class="benefit__heading"
                          >
                            I’m interested in the Red Badger culture, can I get involved?
                          </h4>
                        </div>
                        <div
                          class="benefit__answer__visible"
                        >
                          <p>
                            We’re proud of our culture and we want to consciously grow it as the business expands, join our monthly Culture Committee to help shape our culture conversations.
                          </p>
                        </div>
                      </div>
                    </li>
                    <li>
                      <div>
                        <div
                          class="benefit__question"
                        >
                          <h4
                            class="benefit__heading"
                          >
                            Does Red Badger work with any charities?
                          </h4>
                        </div>
                        <div
                          class="benefit__answer__visible"
                        >
                          <p>
                            We pride ourself on our charity partnerships and we encourage Badgers to get involved in our charity initiatives by joining our Social Value taskforce.
                          </p>
                        </div>
                      </div>
                    </li>
                  </ul>
                </div>
              </li>
              <li>
                <div
                  class="category"
                >
                  <div
                    class="category__content"
                  >
                    <h3
                      class="category__title"
                    >
                      Career development
                    </h3>
                    <img
                      alt="Career development icon"
                      class="category__icon"
                      src="Career_icon.png"
                    />
                  </div>
                  <ul
                    class="category__questionList category__visible"
                  >
                    <li>
                      <div>
                        <div
                          class="benefit__question"
                        >
                          <h4
                            class="benefit__heading"
                          >
                            How can I develop at Red Badger?
                          </h4>
                        </div>
                        <div
                          class="benefit__answer__visible"
                        >
                          <p>
                            Each year you’ll receive a training budget of £2,000 to spend on your professional development. You’ll also be invited to attend various in-house training events and lunch n learns.
                          </p>
                        </div>
                      </div>
                    </li>
                    <li>
                      <div>
                        <div
                          class="benefit__question"
                        >
                          <h4
                            class="benefit__heading"
                          >
                            Do you have a library?
                          </h4>
                        </div>
                        <div
                          class="benefit__answer__visible"
                        >
                          <p>
                            Our HQ office has a well-stocked library, which you are welcome to borrow from or add to. We’ll buy the books on the condition that you share them with everyone!
                          </p>
                        </div>
                      </div>
                    </li>
                    <li>
                      <div>
                        <div
                          class="benefit__question"
                        >
                          <h4
                            class="benefit__heading"
                          >
                            How will I be supported to develop?
                          </h4>
                        </div>
                        <div
                          class="benefit__answer__visible"
                        >
                          <p>
                            You’ll be able to discuss your career development goals with your manager, who will help give you opportunities and support to achieve these.
                          </p>
                        </div>
                      </div>
                    </li>
                  </ul>
                </div>
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </section>
  <div
    class="testimonials"
  >
    <div
      class="container"
    />
  </div>
  <div
    class="videoContainer"
  >
    <div
      class="video"
    >
      <div
        class="container"
      >
        <iframe
          allowfullscreen=""
          class="embed"
          frameborder="0"
          src="https://www.youtube.com/embed/dqJuBdCf-rA"
          title="Red Badger selfie video"
        />
      </div>
    </div>
  </div>
  <div
    class="awardsContainer"
  >
    <div
      class="awards"
    >
      <img
        alt="Awards"
        class="awardsImage"
        sizes="(min-width: 1440px) 1440px, 100vw"
        src="awards-april-2018.jpg"
      />
    </div>
  </div>
</div>
`;
